Automattic/wp-calypso
# ETK: remove coming-soon

## Proposed Changes

ETK: remove coming-soon feature.

Feature migrated to:
https://github.com/Automattic/jetpack/tree/trunk/projects/packages/jetpack-mu-wpcom

Internal, read more at:

* PT: p9dueE-6jY-p2
* FG: PCYsg-Osp-p2

## Testing Instructions

* Coming soon is now running from `jetpack-mu-wpcom` on simple sites. Test the coming soon functionality on a simple site with: `install-plugin.sh editing-toolkit remove/coming-soon-etk on your sandbox`
* Check for any JS or PHP errors.

## Pre-merge Checklist
